src/Configuration.jl

```diff
@@ -128,7 +128,6 @@ Note that Pluto is quickly evolving software, maintained by designers, educators
 end
 
 const RUN_NOTEBOOK_ON_LOAD_DEFAULT = true
-const WORKSPACE_USE_DISTRIBUTED_DEFAULT = true
```

Comment:
Can we continue supporting this feature? It has a lot of cool use cases, and it makes our tests faster!